galaxy_ng/app/access_control/statements/standalone.py

```diff
@@ -94,9 +94,15 @@
         },
         {
             "action": ["update", "partial_update"],
-            "principal": "authenticated",
+            "principal": ["authenticated"],
+            "effect": "allow",
+            "condition": "is_current_user"
+        },
+        {
+            "action": ["update", "partial_update"],
+            "principal": ["group:org_admin"],
```

Comment:
Don't use group principals. This is means that nobody can update any users unless they are `org_admin` group, which is the same as just hardcoding `system:partner-engineers` like we used to do.

Reply:
Good catch. That is a left over from an earlier approach that needs removing.